collect_executables.sh

```diff
@@ -105,6 +105,7 @@ do
     sleep 0.2
 done
 chmod +x geckodriver
+./geckodriver --version
```

Comment:
Is this to ensure the driver exists during runtime?

Reply:
Yeah, I'm hoping it will be useful for debugging when a GHA run crashes.